flooder/src/config.rs

```diff
@@ -21,38 +21,18 @@ pub struct Config {
     #[clap(long, conflicts_with_all = &["phrase"])]
     pub seed: Option<String>,
 
-    /// allows to skip accounts initialization process and just attempt to download their nonces
+    /// allows to skip accounts
     #[clap(long)]
     pub skip_initialization: bool,
 
     /// beginning of the integer range used to derive accounts
     #[clap(long, default_value = "0")]
     pub first_account_in_range: u64,
 
-    /// load transactions from file or generate ad-hoc
```

Comment:
why have we removed many flags?

Reply:
now it is not needed to create a lot of txs before sending them (and removed flags was responsible for storign or loading txs)